src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -48,9 +48,11 @@
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Web" />
+    <Reference Include="System.Xaml" />
```

Comment:
I think system.Xaml should be removed as well, you shouldnt need this in on the model side right?